bika/lims/content/bikasetup.py

```diff
@@ -757,6 +757,45 @@ class PrefixesField(RecordsField):
                           "will be displayed too. By default, disabled.")
         )
     ),
+    BooleanField(
+        'SamplingBarEnabledAnalysisRequests',
+        schemata="Analyses",
+        default=False,
+        widget=BooleanWidget(
+            label=_("Enable the sampling bar in Analysis Requests lists"),
+            description=_(
+                "If you enable this field, the Analysis Requests Lists will"
+                " display an additional filter bar which will allow the user "
+                "to filter the listed items by some terms."
+                "Warning: This may affect the listing performance."),
+        ),
+    ),
+    BooleanField(
+        'SamplingBarEnabledSamples',
+        schemata="Analyses",
+        default=False,
+        widget=BooleanWidget(
+            label=_("Enable the sampling bar in Samples lists"),
+            description=_(
+                "If you enable this field, the Samples Lists will"
+                " display an additional filter bar which will allow the user "
+                "to filter the listed items by some terms."
+                "Warning: This may affect the listing performance."),
+        ),
+    ),
+    BooleanField(
+        'SamplingBarEnabledAnalyses',
+        schemata="Analyses",
+        default=False,
+        widget=BooleanWidget(
+            label=_("Enable the sampling bar in Analyses lists"),
+            description=_(
+                "If you enable this field, the Analyses Lists will"
+                " display an additional filter bar which will allow the user "
+                "to filter the listed items by some terms."
+                "Warning: This may affect the listing performance."),
```

Comment:
Typo: "If enabled, the user will be able to filter the lists of Analyses by several criteria, e.g.:  ......"